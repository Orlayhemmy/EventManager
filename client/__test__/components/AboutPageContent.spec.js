import React from 'react';
import renderer from 'react-test-renderer';
import expect from 'expect';
import AboutPageContent
  from '../../components/AboutPage/Template/AboutPageContent';

describe('AboutPage Component', () => {
  it('should render without throwing error', () => {
    const wrapper = renderer.create(<AboutPageContent />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
