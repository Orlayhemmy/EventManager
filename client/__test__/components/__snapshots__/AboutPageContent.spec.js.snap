// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component should render without throwing error 1`] = `
<div
  className="container"
  id="about"
>
  <div
    className="row"
  >
    <div
      className="media-body bg"
    >
      <br />
      <h1>
        ABOUT US
      </h1>
      <h2>
        What Ecenter Does
      </h2>
      <p>
        eCenter is the leading online venue-finding directory, listing over 1000 event spaces within Nigeria available for exclusive hire. The listings cover the full spectrum of Nigeria event spaces that are available for exclusive hire – for 25 to 5000 guests or delegates – including conference and meetings venues, party venues, product launch venues, halls for hire and wedding ceremony venues. Venue.ng is a free service to users as we earn standard industry commissions from venues for successful introductions via our industry-leading website. Our team are the venue experts who will assist with selecting the best venue shortlist for any event brief. We are based in Lagos, enabling us to show corporate and private event bookers selected venues at short notice and visit new venues as soon as they open.
      </p>
    </div>
  </div>
</div>
`;
